### PR TITLE
Update upgrade.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

**Review by bankovaprgs — APPROVED** [review]
Thank you!

File: implementer-guide/setup/upgrade.md [hunk 1]
```diff
@@ -11,22 +11,45 @@ position: 205
...
+The installer has a step that [backs up the assets' storage]({%slug storage-backup%}) used from the Report Server. This step is optional and turned on by default. We recommend keeping it. On the first run after the upgrade, the Report Server will upgrade the assets storage schema version if necessary. Keep the backed-up assets' storage at least until the Report Server is up and running again.
 
-# After upgrading the Report Server
+## Before upgrading the Report Server
+
+We advise performing a file backup before upgrading. This is specifically recommended if you have changed the application configuration files `/Telerik.ReportServer.Web/Web.config` and `/Services/Telerik.ReportServer.ServiceAgent.exe.config`
+
+### IIS Settings
+
+The _Telerik Report Server_ settings applied in the IIS console, including the HTTPS bindings and AppPool identity will be preserved if you are with version `9.2.23.1115 or later` and upgrade to a newer version.
```
Comment (ighristov) [inline implementer-guide/setup/upgrade.md:29]: "...and AppPool identity will be preserved..."
I think there should be a comma between "identity" and "will be":

File: implementer-guide/setup/upgrade.md [hunk 2]
```diff
@@ -11,22 +11,45 @@ position: 205
...
+
+When upgrading from _Telerik Report Server_ `9.2.23.1010 or older`, you need to reapply the IIS Settings after the upgrade.
+
+This means that if you need automatically preserving the IIS Settings, first you need to upgrade to `9.2.23.1115 or later` and reapply the settings.
+
+__Example__:
+
+* IIS settings will be lost
```
Comment (ighristov) [inline implementer-guide/setup/upgrade.md:37]: Nice explanation 👍

**Review by ighristov — APPROVED** [review]
Thanks for writing this, Toshe! I think I found a small punctuational error, but I might be mistaken.

File: implementer-guide/setup/upgrade.md (hunk 1 above)
Comment (todorarabadzhiev) [reply implementer-guide/setup/upgrade.md]: fixed

**Pull request merged**